app/views/people/edit.html.haml

```diff
@@ -10,12 +10,27 @@
         %ul
           - @person.errors.full_messages.each do |msg|
             %li= msg
-.extra-spacing.row
+
+.row
   .col.s12
-    %h5 Accountgegevens bewerken
-.extra-spacing.row
-  .col.s12
-    = form_for(@person) do |f|
+    = form_for(@person, html: { class: 'settings' }) do |f|
+      %h5 Bankgegevens
+      %p{style: @person.iban.blank? ? 'color:red' : ''}
+        Om de beloning over te maken hebben we nog je IBAN gegevens nodig.
+      %p
+        Dit wordt in een aparte database opgeslagen, zodat de vragen die je net hebt beantwoord alsnog helemaal anoniem zijn.
```

Comment:
Als ik dan ook even naar de inhoud kijk: misschien 'Dit wordt los van de vragenlijsten opgeslagen'? (het wordt niet in een aparte db opgeslagen van de persoonsgegevens die ook in dit formulier staan)

Reply:
Fixed.